polaris-react/src/components/MediaCard/MediaCard.tsx

```diff
@@ -38,6 +38,8 @@ interface MediaCardProps {
    * @default 'medium'
    */
   size?: Size;
+  /** Renders a dismiss `x` button in the top right corner of the card */
```

Comment:
Let's be explicit about what the prop does. The dismiss button and it's position could change and we shouldn't have to worry about updating prop docs
```suggestion
  /** Callback when MediaCard is dismissed */
```